selfdrive/car/hyundai/carstate.py

```diff
@@ -106,10 +106,11 @@ def update(self, cp, cp_cam):
       ret.cruiseState.enabled = cp.vl["TCS13"]["ACC_REQ"] == 1
       ret.cruiseState.standstill = False
     else:
-      ret.cruiseState.available = cp_cruise.vl["SCC11"]["MainMode_ACC"] == 1
+      scc_bus = "SCC12" if self.CP.flags & HyundaiFlags.CAN_CANFD else "SCC11"
```

Comment:
```suggestion
      scc_msg = "SCC12" if self.CP.flags & HyundaiFlags.CAN_CANFD else "SCC11"
```

I checked with cabana for similar bits, but there's not messages I could find with both enabled and available :cry:, besides TCS.